src/components/countrylist.js

```diff
@@ -0,0 +1,30 @@
+import { FaArrowCircleRight } from 'react-icons/fa';
+
+const Country = (props) => {
+  const country = props;
+  // console.log(country.links);
+  return (
+    <div className="wrapper">
+      <div className="btn-head">
+        <h1 className="btn" type="button" id={country.country}>{country.country}</h1>
+        <a className="click-link" href={country.country}>
+          <FaArrowCircleRight />
+        </a>
```

Comment:
- [ ] It would be better if you have the ability to click on the whole card not only on the arrow so I encourage you to do that 
`Hint`: you can achieve that by using `Link`from react-router instead of `<a>` ta